spec/integration/data_stream/spec.yml

```
@@ -78,3 +78,17 @@ spec:
       required: false
       visibility: private
       $ref: "./_dev/spec.yml"
+    - description: File containing routing rules definitions (technical preview)
+      type: file
+      contentMediaType: "application/x-yaml"
+      sizeLimit: 5MB
```

Nit. Are you setting an explicit size limit here for some reason? I think that for `application/x-yaml` files the `configurationSizeLimit` applies, that is already 5MB.
```suggestion
      contentMediaType: "application/x-yaml"
```

---

Just copied from the manifest.yml definition. I'll remove it :+1: